Resources/frontend/js/jquery.adyen-confirm-order.js

```diff
@@ -176,17 +177,19 @@
                             },
                         });
                     },
-                onError: function (error) {
-                    console.log(error);
-                }
+                    onError: function (error) {
+                        console.log(error);
+                    }
                 })
                 .mount('#AdyenChallengeShopperThreeDS2');
         },
 
         handlePaymentDataRedirectShopper: function (data) {
-            var me = this;
-            if (data.action.type === 'redirect') {
-                me.adyenCheckout.createFromAction(data.action).mount(me.opts.mountRedirectSelector);
+            const me = this;
+            if (['redirect', 'qrCode'].includes(data.action.type)) {
```

Comment:
I think this takes more time to read and understand than two equality checks. I'd agree with this change for more items, though.

Reply:
@msilvagarcia simplified `.includes()`